src/CustomerioTracking.tsx

```diff
@@ -143,6 +143,39 @@ class CustomerIO {
     }
     CustomerioReactnative.registerDeviceToken(token)
   }
+
+  /**
+   * Request to show prompt for push notification permissions.
+   * Prompt will only be shown if the current status is - not determined.
+   * In other cases, this function will return current status of permission.
+   * @param options
+   * @returns Success & Failure promises
+   */
+  static async showPromptForPushNotifications(options?: any) : Promise<any>{
+    let pushConfigurationOptions  = {
+      "ios": {
+        "badge" : true,
+        "sound" : true
+      }
+    }
+    if (typeof options !== 'undefined'){
+      pushConfigurationOptions = options
+    }
+  
+  return CustomerioReactnative.showPromptForPushNotifications(pushConfigurationOptions)
+  }
+
+  /**
+   * Get status of push permission for the app
+   * @param handler
+   * @returns Status of push permission as a string
+   */
+  static getPushPermissionStatus(handler? : Function) {
```

Comment:
Do we need this feature? Are customers looking for it?

Reply:
No customer has asked for this but may request in the future. As it was an quick update I managed to do it with this PR itself. There a separate [ticket](https://github.com/customerio/issues/issues/9493) for the same.

Reply:
Ok. Since we have made a ticket for this feature and we believe it's important to add to the SDK, then I approve of adding it here.